src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/NuGetProjectTelemetryService.cs

```diff
@@ -64,21 +64,21 @@ public void EmitNuGetProject(NuGetProject nuGetProject)
 
                 // Get project type.
                 var projectType = NuGetProjectType.Unknown;
-                if (nuGetProject is MSBuildNuGetProject)
+                if (nuGetProject.ProjectStyle == ProjectModel.ProjectStyle.PackagesConfig)
                 {
```

Comment:
I understand, but i find it weird as in same API, we use different mechanisms to check nuget project type, which is little more confusing. Why don't we just use ProjectStyle to check all project types?